modules/client/src/allowlistVoting/internal/methods.ts

```diff
@@ -0,0 +1,470 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import {
+  ContractReceipt,
+  Event as EthersEvent,
+} from "@ethersproject/contracts";
+import { AllowlistVoting } from "../client";
+import {
+  ICreateProposalParams,
+  ISetConfigurationParams,
+  IVoteParams,
+  Steps,
+  Proposal,
+  ProposalCreationValue,
+  VoteStepsValue,
+} from "../interfaces";
+import { arrayify } from "@ethersproject/bytes";
+
+export class AllowlistVotingMethods {
+  private allowlistVoting: AllowlistVoting;
+
+  constructor(allowlistVoting: AllowlistVoting) {
+    this.allowlistVoting = allowlistVoting;
+  }
+
+  /**
+   * Returns the hash for the MODIFY_ALLOWLIST_PERMISSION permission
+   *
+   * @return {*}  {Promise<string>}
+   * @memberof AllowlistVotingMethods
+   */
+  public MODIFY_ALLOWLIST_PERMISSION_ID(): Promise<string> {
+    return this.allowlistVoting
+      .getPluginInstanceWithSigner()
+      .MODIFY_ALLOWLIST_PERMISSION_ID();
+  }
+
+  /**
+   * Returns the PCT_BASE
+   *
+   * @return {*}  {Promise<number>}
+   * @memberof AllowlistVotingMethods
+   */
+  public async PCT_BASE(): Promise<number> {
+    return (
+      await this.allowlistVoting.getPluginInstanceWithSigner().PCT_BASE()
+    ).toNumber();
+  }
+
+  /**
+   * Returns the hash for the SET_CONFIGURATION_PERMISSION permission
+   *
+   * @return {*}  {Promise<string>}
+   * @memberof AllowlistVotingMethods
```

Comment:
Sae with the rest of the permissions, we must choose either one or the other, we must also take into account that if we switch we will break the UI

Reply:
done